lock-impl/src/test/java/com/palantir/lock/logger/LockServiceStateLoggerTest.java

```diff
@@ -39,9 +37,6 @@
 import com.palantir.lock.LockRequest;
 import com.palantir.lock.SimpleTimeDuration;
 import com.palantir.lock.StringLockDescriptor;
-import com.palantir.lock.impl.ClientAwareReadWriteLock;
-import com.palantir.lock.impl.LockClientIndices;
-import com.palantir.lock.impl.LockServerLock;
 import com.palantir.lock.impl.LockServiceImpl;
 
 public class LockServiceStateLoggerTest {
```

Comment:
it is a smoke test, to make sure logging doesn't fail. I use it to see the result on disk and check if it is ok. We can add to make the check automatic.